ttnn/cpp/ttnn/operations/data_movement/slice/device/slice_op.cpp

```diff
@@ -94,18 +98,39 @@ void SliceDeviceOperation::validate_with_output_tensors(
             "Can only unpad tilized tensor with full tiles");
     } else if (input_tensor_a.get_layout() == Layout::ROW_MAJOR) {
         TT_FATAL(
-            (output_tensor_shape[-1] * input_tensor_a.element_size() % sizeof(uint32_t) == 0) &&
-                (this->slice_start[-1] * input_tensor_a.element_size() % sizeof(uint32_t) == 0),
+            (output_tensor_shape[-1] * input_tensor_a.element_size() % sizeof(uint32_t) == 0),
             "RM unpadding requires output X size to be packable");
+        if (this->step.has_value()) {
```

Comment:
It seems like the wording could be a bit better here.
What about something more verbose like...
"An unpadding slice operations for a RowMajor layout on the output tensor requires the last dimension to be on a 32 bit boundary.  For example, the final dimension needs to be divisible by 2 for bfloat16 or divisible by 4 for bfloat8"

Reply:
Done.